test/model.validate.test.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
+
+const start = require('./common');
+
+const assert = require('assert');
+
+const mongoose = start.mongoose;
+const Schema = mongoose.Schema;
+
+describe('model: validate: ', function() {
+  afterEach(() => mongoose.deleteModel(/.*/));
```

Comment:
```suggestion
  beforeEach(() => mongoose.deleteModel(/.*/));
```

change it to `before` because from what i know mocha re-uses the same environment, which means it has the possibility of having models transfer from other tests here and conflicting
also it is what is used in other tests